network/player_test.go

```diff
@@ -69,3 +65,117 @@ func TestConvertToCreatePlayerResponse(t *testing.T) {
 		assert.Equal(t, tc.expResp, resp, tc.desc)
 	}
 }
+
+func TestConvertToGetActiveGamesForPlayerResponse(t *testing.T) {
```

Comment:
If we put Alice in all the games, shouldn’t we also test games without Alice? Is this supposed to filter games only including the player, or is the assumption that the games passed in are already filtered?